map_model/src/objects/traffic_signals.rs

```diff
@@ -404,14 +408,27 @@ impl Stage {
         }
     }
 
-    // trivial function that returns true if the stage is just crosswalks
-    pub fn contains_only_crosswalks(&self) -> bool {
+    // A trivial function that returns max crosswalk time if the stage is just crosswalks.
+    pub fn contains_only_crosswalks(
```

Comment:
The name really makes it sound like this returns a bool. How about `max_crosswalk_time` and keep the docstring, which explains it's only if everything's a crosswalk?